README.md

```diff
@@ -24,12 +24,11 @@ npm install mime-db
 ### Database Download
 
 If you're crazy enough to use this in the browser, you can just grab the
-JSON file using [RawGit](https://rawgit.com/). It is recommended to replace
-`master` with [a release tag](https://github.com/jshttp/mime-db/tags) as the
-JSON format may change in the future.
+JSON file using [unpkg](https://unpkg.com/mime-db/db.json). It is recommended 
+to keep the version in the URL as the JSON format may change in the future.
```

Comment:
Can you add instructions or a link to instructions on how to actually put the version in the URL? Based on this sentence and the given example I truthfully have no idea how to put the version in there.

Reply:
Understand. I've updated the instructions now